flask_resources/resources.py

```diff
@@ -32,6 +33,7 @@ class ResourceConfig:
     item_route = "/resources/<id>"
     list_route = "/resources/"
     request_url_args_parser = ArgsParser()
+    error_mapper = ErrorMapper
```

Comment:
Can we map this to an instance (or a function) to be consistent with the other configurations?

Reply:
No, because this is used as a decorator, so when it will be initialized it will take the wrapped function as a parameter or otherwise I would need to create a `wrap` method to set the function to wrap on runtime...

```
class ErrorMapper:

    def wrap(self, func):
        update_wrapper(self, func)
        self.func = func

def error_mapper(f):
    ...
    mapped_f = _error_mapper.wrap(f) if _error_mapper else f
    ...
```

@lnielsen which one do you prefer?

Reply:
If it's a class, it should be ``error_mapper_cls``, if it's instance/function ``error_mapper``.